x-pack/plugins/security_solution/public/management/links.test.ts

```diff
@@ -230,6 +235,18 @@ describe('links', () => {
     expect(filteredLinks).toEqual(getLinksWithout(SecurityPageName.eventFilters));
   });
 
+  it('should hide Blocklist for user without privilege', async () => {
```

Comment:
Should we add a test case for the case it has right privileges and Blocklist is shown?

Reply:
There is a test case for showing all links, I believe that should be enough. What was missing is the correct name, so I renamed it: 00e8553